pylops/waveeqprocessing/twoway.py

```diff
@@ -245,17 +247,6 @@ def _srcillumination_oneshot(self, isrc: int) -> Tuple[NDArray, NDArray]:
             Source illumination
 
         """
-        # create geometry for single source
-        geometry = AcquisitionGeometry(
-            self.model,
-            self.geometry.rec_positions,
-            self.geometry.src_positions[isrc, :],
-            self.geometry.t0,
-            self.geometry.tn,
-            f0=self.geometry.f0,
-            src_type=self.geometry.src_type,
-        )
-        solver = AcousticWaveSolver(self.model, geometry, space_order=self.space_order)
 
         # assign source location to source object with custom wavelet
         if hasattr(self, "wav"):
```

Comment:
To be consistent with the `_born_oneshot` method, shall we make this 

```
self.wav.coordinates.data[0, :] = solver.geometry.src_positions[:]
```

?

Reply:
You're right! I'll make the changes and update the PR.